### DISCUSSION TITLE
Speaker for CYD

### DISCUSSION THREAD

**Discussion by 88markus88** [witnessmenow/ESP32-Cheap-Yellow-Display#99] [Q&A]
I cannot find information about the speaker port of the CYD. According to the PIN list it is connected via IO26. But: Is it simply connected to GPIO26 to be used in DAC mode of the ESP32, or is there an amplifier? Is ther an example how to use it?
Thanks for any help,
Markus

**Comment by peteDDD** [comment]
I want to get this working too.  Here are some links I am looking at which should help

https://www.keyestudio.com/products/keyestudio-sc8002b-audio-power-amplifier-speaker-module-for-arduino-player

https://wiki.keyestudio.com/KS0374_keyestudio_Power_Amplifier_Module

https://docs.arduino.cc/tutorials/generic/simple-audio-player/

**Reply by peteDDD** [reply]
THIS: https://www.youtube.com/watch?v=oZ39VCUvKjw&list=PL5vDt5AALlRfGVUv2x7riDMIOX34udtKD&index=3
play wav files ...

**Comment by clach04** [comment]
I have speaker working for tones (in micropython)... but the screen flashes off when doing so 😭 

Anyone else seeing this?

https://github.com/clach04/cyd_clocks/blob/main/rainbow_test_speaker.py

It seems worse when using nano-GUI
